source/base/Text.ooc

```diff
@@ -174,6 +174,14 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	print: func {
+		printf("%s", this _buffer _backend pointer as Char*)
```

Comment:
That's interesting. Does the process on android system have a `stdout` stream attached ? I mean, can you do `fwrite(stdout, "my string\n")` instead of `println` ?